book/animations.md

```diff
@@ -2119,7 +2128,7 @@ each display item:
 [^clipping-notes]: This is very important, because otherwise some
 composited layers can end up huge despite not drawing much to the screen.
 A good example of this optimization making a big difference is loading the
-browser from [Chapter 15](embeds.md) for the `browser.engineering` homepage,
+browser from [Chapter 15](embeds.md) for the browser.engineering homepage,
```

Comment:
I think if we write `http://browser.engineering/` it'll render a hyperlink in builds of the book where that's possible.